dataprofiler/labelers/column_name_model.py

```diff
@@ -0,0 +1,248 @@
+"""Contains class for column name data labeling model."""
+import json
+import os
+
+import numpy as np
+
+try:
+    from rapidfuzz import fuzz, process
```

Comment:
I'm wondering if we should just import `rapidfuzz` bc otherwise down below we don't really know where `fuzz` and especially `process` come from. the latter being extremely generic.

Reply:
totally fair -- great point